runtime-decompiler/src/main/java/org/jrd/frontend/frame/main/GlobalConsole.java

```diff
@@ -32,7 +32,12 @@ public GlobalConsole() {
                 tmpFrame = new JFrame();
                 tmpFrame.setLayout(new BorderLayout());
                 tmpFrame.add(new JScrollPane(tmpLog));
-                tmpFrame.add(tmpClean, BorderLayout.SOUTH);
+                JPanel p = new JPanel(new BorderLayout());
+                JCheckBox verbose = new JCheckBox("print whatever nonsense", Logger.getLogger().isVerbose());
```

Comment:
`"print whatever nonsense"`, lol. Please replace this with a more *appropriate* String, like `"Enable verbose logging"`.